src/snowflake/cli/_plugins/streamlit/manager.py

```diff
@@ -57,6 +58,10 @@ def _put_streamlit_files(
         stage_manager = StageManager()
         for file in artifacts:
             if file.is_dir():
+                if not bool(os.listdir(file)):
+                    cli_console.warning(f"Skipping empty directory: {file}")
+                    continue
```

Comment:
Maybe I should add it in `StageManager.put`? WDYT?

Reply:
IMHO it belongs here- it's streamlit logic, not stage put logic- it's the Streamlit manager that should decide what files and dirs to put, and Stage manager should know how to handle them to put'em on stage.